app/controllers/api/v0/admin/auctions_controller.rb

```diff
@@ -8,5 +8,7 @@ def index
   def show
     auction = Auction.find(params[:id])
     render json: auction, serializer: ::Admin::AuctionSerializer
+  rescue ActiveRecord::RecordNotFound
```

Comment:
maybe we can put all of these rescues into the `api_controller` like  http://stackoverflow.com/a/24451181/1019369 ?